### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 171164558**
Update a paper of event-based neural radiance fields (ICCV 2023)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1209,6 +1212,9 @@ European Conference on Computer Vision (ECCV), 2022. [PDF](https://arxiv.org/pdf
 - <a name="Chen23icvr"></a>Chen, H., Chung, V., Tan, L., Chen, X.,  
 *[Dense Voxel 3D Reconstruction Using a Monocular Event Camera](https://doi.org/10.1109/ICVR57957.2023.10169359)*,  
 Int. Conf. Virtual Reality (ICVR), 2023. [PDF](https://arxiv.org/pdf/2309.00385), [Dataset](https://www.kaggle.com/datasets/hche8927/synthevox3d).
+- <a name="Qi23iccv"></a>Qi, Y., Zhu, L., Zhang, Y., Li, J.,
```
Comment (guillermogb) [inline README.md:1215]: hello. there cannot be two entries as primary ones. Please see other examples, like Bardow's CVPR 2016 paper.

**Review by guillermogb — CHANGES_REQUESTED** [review]

**Comment by 171164558** [comment]
Sorry for the mistake. I have deleted the duplicated entry and kept the entry that best fits the category.

**Comment by guillermogb** [comment]
you can have a secondary entry... that is why I mentioned the example in Bardow CVPR 16. The secondary entry should be a "pointer" to the first entry.

**Comment by 171164558** [comment]
Thank you for your guidance,  Gallego. The secondary entry has been added as Bardow CVPR 16.

**Review by guillermogb — APPROVED** [review]
thanks

**Pull request merged**